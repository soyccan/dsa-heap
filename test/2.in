100 100
add 12 39 99 
drop 29 
quota 68 49 
add 0 48 85 
quota 52 74 
quota 86 93 
drop 17 
quota 72 63 
quota 99 54 
drop 88 
drop 19 
add 78 59 51 
quota 40 56 
quota 41 44 
drop 41 
drop 23 
quota 56 64 
print 2 
drop 97 
print 79 
add 89 30 18 
print 71 
quota 82 23 
print 93 
print 40 
drop 35 
drop 27 
add 36 69 14 
print 62 
drop 24 
drop 76 
drop 91 
add 61 3 93 
add 22 56 67 
print 38 
drop 70 
add 69 63 18 
quota 53 45 
quota 89 14 
print 0 
quota 25 24 
print 2 
print 62 
drop 20 
quota 3 16 
print 59 
print 98 
quota 26 21 
quota 95 36 
print 52 
quota 97 89 
drop 63 
add 30 40 70 
print 63 
add 14 77 34 
drop 95 
quota 55 86 
add 38 85 97 
add 58 31 12 
drop 82 
add 83 31 58 
quota 95 2 
print 87 
add 50 51 48 
add 28 82 55 
print 71 
drop 62 
print 0 
print 21 
quota 57 52 
quota 3 86 
quota 86 18 
print 51 
quota 10 46 
add 34 97 3 
print 65 
add 17 72 59 
quota 82 73 
add 35 24 85 
add 33 37 37 
add 76 86 74 
drop 94 
quota 59 4 
add 11 90 21 
print 25 
drop 51 
print 11 
drop 36 
quota 88 0 
add 12 86 36 
drop 75 
quota 34 51 
add 9 97 58 
drop 56 
drop 10 
add 4 83 35 
drop 74 
drop 77 
quota 52 13 
drop 40 
