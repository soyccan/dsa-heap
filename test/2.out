12 added to printer 99 with priority 39
no jobs in 29
quota 68 set for 49
0 added to printer 85 with priority 48
quota 52 set for 74
quota 86 set for 93
no jobs in 17
quota 72 set for 63
quota 99 set for 54
no jobs in 88
no jobs in 19
78 added to printer 51 with priority 59
quota 40 set for 56
quota 41 set for 44
no jobs in 41
no jobs in 23
quota 56 set for 64
no jobs in 2
no jobs in 97
no jobs in 79
89 added to printer 18 with priority 30
no jobs in 71
quota 82 set for 23
no jobs in 93
no jobs in 40
no jobs in 35
no jobs in 27
36 added to printer 14 with priority 69
no jobs in 62
no jobs in 24
no jobs in 76
no jobs in 91
61 added to printer 93 with priority 3
22 added to printer 67 with priority 56
no jobs in 38
no jobs in 70
69 added to printer 18 with priority 63
quota 53 set for 45
quota 89 set for 14
no jobs in 0
quota 25 set for 24
no jobs in 2
no jobs in 62
no jobs in 20
quota 3 set for 16
no jobs in 59
no jobs in 98
quota 26 set for 21
quota 95 set for 36
no jobs in 52
quota 97 set for 89
no jobs in 63
30 added to printer 70 with priority 40
no jobs in 63
14 added to printer 34 with priority 77
no jobs in 95
quota 55 set for 86
38 added to printer 97 with priority 85
58 added to printer 12 with priority 31
no jobs in 82
83 added to printer 58 with priority 31
quota 95 set for 2
no jobs in 87
50 added to printer 48 with priority 51
28 added to printer 55 with priority 82
no jobs in 71
no jobs in 62
no jobs in 0
no jobs in 21
quota 57 set for 52
quota 3 set for 86
quota 86 set for 18
1879048192 printed on printer 51
quota 10 set for 46
34 added to printer 3 with priority 97
no jobs in 65
17 added to printer 59 with priority 72
quota 82 set for 73
35 added to printer 85 with priority 24
33 added to printer 37 with priority 37
76 added to printer 74 with priority 86
no jobs in 94
quota 59 set for 4
11 added to printer 21 with priority 90
no jobs in 25
no jobs in 51
no jobs in 11
no jobs in 36
quota 88 set for 0
12 added to printer 36 with priority 86
no jobs in 75
quota 34 set for 51
9 added to printer 58 with priority 97
no jobs in 56
no jobs in 10
4 added to printer 35 with priority 83
0 dropped on printer 74
no jobs in 77
quota 52 set for 13
no jobs in 40
